qubesbuilder/plugins/github/webhooks/webhooks.conf

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
I'm not ok that you delete this file because there exists other services like https://github.com/fepitre/qubes-g2g-continuous-integration/blob/master/webhooks/gitlabci.py using this webhooks code and it's easier to register them into this configuration file.

Reply:
Reverted